package.json

```diff
@@ -39,6 +39,10 @@
     "decaffeinate-coffeescript": "^1.10.0-patch5",
     "decaffeinate-parser": "^2.0.2",
     "detect-indent": "^4.0.0",
+    "eslint-config-standard": "^6.2.1",
+    "eslint-plugin-html": "^1.5.5",
+    "eslint-plugin-promise": "^3.3.0",
+    "eslint-plugin-standard": "^2.0.1",
```

Comment:
"extends": "eslint:recommended"

it means to use esint-config-standard
and it depends those 3 plugins